src/database-controller/sdk/index.js

```diff
@@ -125,6 +125,48 @@ class DatabaseModel {
       },
     );
 
+    class TaskHistory extends Model {}
```

Comment:
Also rename previous taskroleindex to taskindex?

Reply:
I prefer to fix it in a different PR. Because it is not trivial, and needs some codes to handle the legacy